agent.go

```diff
@@ -1295,7 +1295,7 @@ func newContainerCb(pod *pod, data []byte) error {
 		return fmt.Errorf("Container %s already exists, impossible to create", payload.ID)
 	}
 
-	absoluteRootFs, err := mountContainerRootFs(payload.ID, payload.Image, payload.RootFs, payload.FsType)
+	absoluteRootFs, err := mountContainerRootFs(payload.ID, payload.Image, payload.RootFs, payload.FsType, payload.SCSIAddr)
```

Comment:
We should pass the payload to this function, instead of extending its number of parameters, all being attributes of the same structure.

Reply:
I had done this to avoid introducing hyperstart specific structures in syscall.go.

Reply:
There is no harm doing so. `syscall.go` is part of the agent, which makes it tied to hyperstart. I don't think this is an issue to pass an hyperstart structure to this function, and this would simplify things as I mentioned.

Reply:
I was just trying to avoid introducing any hyperstart structures here, as this is more like an utils package and is a good contender for moving into its own util package, but that is for the future. 
I can simplify this for now.

Reply:
I have pushed changes for this.